NSS-Evening-E22/pos-system-outlaw-bandanas
# Sb item edit

## Description
User is able to click Edit button on Item card, item Add/Edit form loads and user can edit item name or price.  Update saves adn returns user to Order Details screen.

## Related Issue
#17 

## Motivation and Context
Requirement that user can edit item details.

## How Can This Be Tested?
login to app
Click View Orders
Click Details on order
Click Edit on item
modify item property
click Add/Edit button
ensure change is reflected on item in order details


## Screenshots (if appropriate):

## Types of changes

- [x ] New feature (non-breaking change which adds functionality)
